content/en/blog/general/jmespath/index.md

```diff
@@ -0,0 +1,63 @@
+### Time_diff
+
+The `time_diff()` filter is used to find how much time has passed between two time period. For example, `time_diff('2023-01-10T00:00:00Z','2023-01-11T00:00:00Z')` results in the value `"24h0m0s"`. It has been added for a common usecase where we want to check, whether a certain amount of time has been passed since an important event or not, such as to check whether 4 hours have passed since last scan was finished, we can use `time_diff()` filter like this `"{{ time_diff('{{metadata.scanFinishedOn}}','{{ time_now_utc() }}') }}"`
+### Time_now
+
+The `time_now()` filter, as the name suggests returns the current time. It has been introduced for the common case where we need current time as an argument for other filters. We require current time in several filters like `time_add()` and this is added for such cases.
+
+### Time_now_utc
+
+The `time_now_utc()` filter works similar to `time_now()` where it returns the current time according to UTC. It has been introduced for the common case where we need current time as an argument for other filters.
+### Time_parse
+
+The `Time_parse()` filter is an interesting addition where we can pass any time string in any format to convert it ino the Kyverno standard RFC 3339 format. It takes 2 arguments, the time format and the time string. For example, `time_parse('Mon Jan 02 2006 15:04:05 -0700', 'Fri Jun 22 2022 17:45:00 +0100')` results in the output of `"2022-06-22T17:45:00+01:00"`. This field has been added to allow standardization such that all the timestamps added as metadata and annotations follow the RFC 3339 format.
+
+### Time_to_cron
+
+In kyverno, we use cron in several places to perform job scheduling. The cron expression enables users to schedule tasks to run periodically at a specified date/time. And it's naturally a great tool for automating lots of process runs, which otherwise would require human intervention. We have added a `time_to_cron()` filter to generate a cron expression from a time. For example, `time_to_cron('2022-04-11T03:14:05-07:00')` results in the output `"14 3 11 4 1"`. This filter can be used to generate things like ClusterCleanupPolicy.
+
+### Time_truncate
+The `time_truncate()` filter is used to round the given time to the nearest duration which is specified in the arguments. For example,  `time_truncate('2023-01-12T17:37:00Z','1h')` results in the output `"2023-01-12T17:00:00Z"`. 
+
+### Time_utc
+#### Description
+The `time_utc()` filter takes in a time in RFC 3339 format with a time offset and presents the same time in UTC/Zulu. It can be used in pair with `time_parse()` to convert it to UTC.
```

Comment:
What I mean is all of this. Blogs don't need to list all the filters out. This is what documentation is for. Blogs tell a story around a use case in a casual style.